### PR TITLE
feat: remove color in constants

### PR DESCRIPTION
**Pull request by Zero-Rock**
## What this PR does / why we need it:

remove color in constants

## I have checked the following points:
- [X] I18n is finished and updated by cli
- [X] Form fields validation is added and length is limited
- [X] Display normally on small screen
- [X] Display normally when some data is empty or null
- [X] Display normally in english mode

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | use color in theme-color |
| 🇨🇳 中文    | 使用 theme-color 中的颜色  |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/theme-color.mjs [hunk 1]
```diff
@@ -23,16 +23,16 @@ const colors = {
   red: '#d84b65',
   'red-1': 'rgba(216,75,101,0.1)',
   danger: 'rgba(216,75,101,1)',
-  error: 'rgba(216,75,101,1)',
+  error: '#d84b65',
```
Comment (daskyrk) [inline shell/app/theme-color.mjs:26]: Should keep rgba type, you can add `d84b65 ` in comment.

Comment (Zero-Rock) [reply shell/app/theme-color.mjs]: these colors are used in some places as arguments To the function colorToRgb(color: string, opacity?: number), and can only be of hexadecimal type

**Comment by daskyrk** [comment]
Fill in changeLog

**Pull request merged**